### PR TITLE
First android prototype with our model. Issue #26.

### PR DESCRIPTION
**Pull request by jankejc**
- Branch contains whole app prototype with our model.
- It also contains the way to add metadata.
- Branch does not contain `build` folder because it was too large.
- Branch closes issue #26.

### CODE REVIEW

**Review by qaziok — DISMISSED** [review]
Gites że jest - niech jeszcze @kjedrasz2137 klepnie czy wszystko się zgadza

File: Android/adding_metadata/readme.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSTRUCTION TO ADD METADATA
+
+1. For me, the most suitable tool to build environment for the script was Anaconda.
+2. After installation od all packages go to current folder in terminal.
+3. In file we are using mean = 0 and std = 1. That's because our model is already normalized, so we need to add metadata only.
```
Comment (kjedrasz2137) [inline Android/adding_metadata/readme.txt:5]: Our model has a normalization layer inside, i.e. `layers.Rescaling()`, that is why we can set mean = 0 and std = 1 in the metadata file.

File: Android/adding_metadata/readme.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+5. Don't forget to add .tflite model to folder and change the input and output in script in order to use it by yourself. 
+4. In the bottom I have added the example of running python script. Running it from VScode was too problematic for me.
+
+python ./metadata_writer_for_image_classifier.py --model_file=./model_1_0.tflite --label_file=./labels.txt --export_directory=./model_with_metadata/python ./metadata_writer_for_image_classifier.py --model_file=./model_1_0.tflite --label_file=./labels.txt --export_directory=model_with_metadata
```
Comment (kjedrasz2137) [inline Android/adding_metadata/readme.txt:9]: If we use folder `model_with_medata`, maybe we can also use a similar folder called `model_without_metada`?

File: Android/adding_metadata/metadata_writer_for_image_classifier.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+_MODEL_INFO = {
+    "model_1_0.tflite":
+        ModelSpecificInfo(
+            name="Polish Banknotes image classifier",
+            version="v2",
+            image_width=224,
+            image_height=224,
+            image_min=0.0,
+            image_max=1.0,
```
Comment (kjedrasz2137) [inline Android/adding_metadata/metadata_writer_for_image_classifier.py:70]: `image_min` should be 0 and `image_max` should be 255

File: Android/adding_metadata/metadata_writer_for_image_classifier.py [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+            mean=[0],
```
Comment (kjedrasz2137) [inline Android/adding_metadata/metadata_writer_for_image_classifier.py:71]: Please set mean = [0.0], std = [1.0]

File: Android/adding_metadata/labels.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+10
+20
+50
+100
+200
+500
+none
```
Comment (kjedrasz2137) [inline Android/adding_metadata/labels.txt:7]: Should be `None`

File: Android/PolishBanknotesApp/app/src/main/java/org/tensorflow/lite/examples/imageclassification/fragments/CameraFragment.kt [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+            if (imageClassifierHelper.maxResults > 1) {
+                imageClassifierHelper.maxResults--
+                updateControlsUi()
+                classificationResultsAdapter.updateAdapterSize(size = imageClassifierHelper.maxResults)
+            }
+        }
+
+        // When clicked, increase the number of objects that can be classified at a time
+        fragmentCameraBinding.bottomSheetLayout.maxResultsPlus.setOnClickListener {
+            if (imageClassifierHelper.maxResults < 3) {
```
Comment (kjedrasz2137) [inline Android/PolishBanknotesApp/app/src/main/java/org/tensorflow/lite/examples/imageclassification/fragments/CameraFragment.kt:164]: Compare `imageClassifierHelper.maxResults` with number of classes, which in our case is 7.

**Review by kjedrasz2137 — CHANGES_REQUESTED** [review]
Please read my comments and make the necessary corrections.

**Review by jankejc — COMMENTED** [review]
All the requested changes have been applied.

**Review by kjedrasz2137 — APPROVED** [review]

**Comment by kjedrasz2137** [comment]
Looks good

**Pull request merged**